src/test/java/de/qabel/desktop/CellValueFactory/BoxObjectCellValueFactoryTest.java

```diff
@@ -0,0 +1,107 @@
+package de.qabel.desktop.CellValueFactory;
```

Comment:
most of this file seems to be a duplication of `BoxFileCellValueFactoryTest.java`.
Either both classes should be merged or a superclass should be extracted like `AbstractCellValueFactoryTest`.